src/SourceLocator/Type/Composer/Factory/MakeLocatorForComposerJson.php

```diff
@@ -26,6 +26,15 @@
 use function json_decode;
 use function realpath;
 
+/**
+ * @psalm-type ComposerAudoload array{
```

Comment:
Typo `ComposerAudoload` => `ComposerAutoload`.

Anyway the types from https://github.com/composer/composer/pull/10516 will ship with the next Composer 2.2 patch release if you want to start relying on that instead.

Reply:
Good catch! Still, adding a dependency for just the type alias is a bit much, in this case

Reply:
Yeah I thought maybe it could be a dev dependency as you only need it for static analysis, but I'm not sure if the APIs are exposed publicly here, if that's the case then yes please duplicate rather than depending on composer/composer, that always ends up horribly.